# HG changeset patch
# User joe
# Date 1542505743 25200
#      Sat Nov 17 18:49:03 2018 -0700
# Node ID e760c9dd6eec4d5922a44079d9e633d7242eae54
# Parent  834044099a6bf361c98b7b4824a3e3eaede66630
increase the tcp window scale factor to 14 from 3

diff -r 834044099a6b -r e760c9dd6eec sys/src/9/ip/tcp.c
--- a/sys/src/9/ip/tcp.c	Fri Nov 16 10:55:45 2018 -0700
+++ b/sys/src/9/ip/tcp.c	Sat Nov 17 18:49:03 2018 -0700
@@ -43,7 +43,7 @@
 	MSSOPT		= 2,
 	MSS_LENGTH	= 4,		/* Maximum segment size */
 	WSOPT		= 3,
-	WS_LENGTH	= 3,		/* Bits to scale window size by */
+	WS_LENGTH	= 14 /*3*/,		/* Bits to scale window size by */
 	MSL2		= 10,
 	MSPTICK		= 50,		/* Milliseconds per timer tick */
 	DEF_MSS		= 1460,		/* Default maximum segment */
